### PR TITLE
feat:add modal component

### PR DESCRIPTION
**Pull request by mariyasada**
adding a modal feature to CL. do review it. adding modal file link to all files and

### CODE REVIEW

**Comment by shaksham08** [comment]
Good Work! Looks Good 👍

File: stylesheets/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    align-items: center;
+}
+
+.modal-container
+{
+    height: auto;
+    max-width:40vw;
+    background:linear-gradient(var(--light-teal),var(--white-color));
+    box-shadow: 2px 2px 4px var(--ex-light-purple);
+    /* color:var(--white-color); */
```
Comment (shaksham08) [inline stylesheets/modal/modal.css]: This extra comment can be removed

File: stylesheets/modal/modal.css [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
 }
-
 .modal-container
 {
     height: auto;
-    max-width:40vw;
+    min-width:40vw;
     background:linear-gradient(var(--light-teal),var(--white-color));
     box-shadow: 2px 2px 4px var(--ex-light-purple);
     /* color:var(--white-color); */
```
Comment (Bhakti199) [inline stylesheets/modal/modal.css]: remove extra comment

**Review by Bhakti199 — COMMENTED** [review]
you have structured the code nicely.

File: stylesheets/modal/modal.css (hunk 2 above)
Comment (mariyasada) [reply stylesheets/modal/modal.css]: okay. done

**Review by shaksham08 — APPROVED** [review]

**Review by Bhakti199 — APPROVED** [review]

File: stylesheets/modal/modal.css (hunk 1 above)
Comment (mariyasada) [reply stylesheets/modal/modal.css]: will remove this

**Pull request merged**